### DISCUSSION TITLE
How to install on a single bare metal?

### DISCUSSION THREAD

**Discussion by dhlao** [okd-project/okd#1547] [Q&A]
After reading the online documents on [https://docs.okd.io/latest/installing/installing_bare_metal_ipi/ipi-install-overview.html](url), I realise that the installation process requires at least 3 machines (Provisioner, Control Panel node, and Worker node). However, all I have is just one physical bare metal. Creating additional machines (physical or virtual) in the same network is not possible in my situation.

I previously used oVirt node as a hypervisor for testing purposes and so no HA is needed. The procedure is simple, boot the installation media and kick off the installation. At the end, logging in the web console and starting to create new VMs. This type of setup is quite common such as Citrix XenServer and VMware ESXi.

It seems that "Openshift Assisted Installer" is designed to simplify the installation as well. I wonder if OKD also has this feature.

**Comment by vrutkovs** [comment]
Yes - https://github.com/openshift/assisted-service/tree/master/deploy/podman#okd-configuration

We removed it from the docs since it requires deploying your own Assisted Service and the docs page mentioned console.redhat.com - see https://github.com/openshift/openshift-docs/issues/51152 and https://github.com/okd-project/planning/issues/27

You don't necessarily need assisted installer though - see https://github.com/openshift/enhancements/blob/master/enhancements/installer/single-node-installation-bootstrap-in-place.md - however it would be easier to handle

**Reply by dhlao** [reply]
Thanks for the information. Since the idea of OKD installation without additional nodes is still at the planning stage, I need to keep using oVirt although its future is uncertain owing to the discontinuance of RHV.
